apps/examples/testcase/ta_tc/media/utc/utc_media_mediaplayer.cpp

```diff
@@ -478,6 +478,32 @@ static void utc_media_MediaPlayer_setVolume_n(void)
 	mp.destroy();
 }
 
+static void utc_media_MediaPlayer_isPlaying_p(void)
+{
+	media::MediaPlayer mp;
+	std::unique_ptr<media::stream::FileInputDataSource> source = std::move(std::unique_ptr<media::stream::FileInputDataSource>(new media::stream::FileInputDataSource(dummyfilepath)));
+	mp.create();
+	mp.setDataSource(std::move(source));
+	mp.prepare();
+	mp.start();
+
+	TC_ASSERT_EQ("utc_media_MediaPlayer_isPlaying", mp.isPlaying(), true);
+
+	mp.unprepare();
+	mp.destroy();
+
+	TC_SUCCESS_RESULT();
+}
+
+static void utc_media_MediaPlayer_isPlaying_n(void)
+{
+	media::MediaPlayer mp;
+
```

Comment:
could you add one case~? after mp.create()
Substantially, that is a return to what was not mp create.

Reply:
I added `after mp.create()` test case